aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/UpdateHandler.java

```diff
@@ -152,6 +152,23 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                                 updateAssociatedRoles(proxy, rdsClient, progress, previousRoles, desiredRoles),
                         CallbackContext::isUpdatedRoles, CallbackContext::setUpdatedRoles)
                 )
+                .then(progress -> Commons.execOnce(progress, () -> {
+                            if (ResourceModelHelper.shouldStopAutomaticBackupReplication(request.getPreviousResourceState(), request.getDesiredResourceState())) {
+                                final DBInstance dbInstance = fetchDBInstance(rdsProxyClient.defaultClient(), progress.getResourceModel());
+                                return stopAutomaticBackupReplicationInRegion(dbInstance, proxy, progress, rdsProxyClient.defaultClient(),
+                                        ResourceModelHelper.getAutomaticBackupReplicationRegion(request.getPreviousResourceState()));
+                            }
+                            return progress;},
+                        CallbackContext::isAutomaticBackupReplicationStopped, CallbackContext::setAutomaticBackupReplicationStopped))
+                .then(progress -> Commons.execOnce(progress, () -> {
+                            if (ResourceModelHelper.shouldStartAutomaticBackupReplication(request.getPreviousResourceState(), request.getDesiredResourceState())) {
+                                final DBInstance dbInstance = fetchDBInstance(rdsProxyClient.defaultClient(), progress.getResourceModel());
+                                return startAutomaticBackupReplicationInRegion(dbInstance, proxy, progress, rdsProxyClient.defaultClient(),
+                                        ResourceModelHelper.getAutomaticBackupReplicationRegion(request.getDesiredResourceState()));
+                            }
+                            return progress;
+                        },
+                        CallbackContext::isAutomaticBackupReplicationStarted, CallbackContext::setAutomaticBackupReplicationStarted))
```

Comment:
Given these blocks are mutually exclusive, we might consider joining them in a single routine

Reply:
They are not mutually exclusive, we need to first stop replication and then start it in a new region. So if replication is changed from `eu-west-1` region to `eu-west-2` two calls are expected.